frappe/public/js/frappe-web.bundle.js

```diff
@@ -21,6 +21,7 @@ import "./frappe/model/model.js";
 import "./frappe/model/perm.js";
 
 import "./bootstrap-4-web.bundle";
+import hljs from "./syntax_highlight";
```

Comment:
duplicate line?

Reply:
Under the auspice that I may not have understood your comment fully:

This part is an isomertic transformation ("true" refactoring). Previously it was part of the import closure from `import "../../website/js/website.js";`, now that it isn't, we must ensure the closure on the calling site here in order to not change behavior of `frappe-web.bundle.js`.

I interpret your question as: "why not drop this entirely?" - I can't really judge with authority, but if I reproduce a correct interpretation of @ankush 's comment on that question, it might be a valid follow up PR.